distribution/changelog.txt

```diff
@@ -4,6 +4,7 @@
 - Improved: [#21769] Expose “animation is backwards” wall property in Tile Inspector.
 - Change: [#21715] [Plugin] Remove access to the internal `owner` property. Note: `ownership` is still accessible.
 - Fix: [#866] Boat Hire boats get stuck entering track.
+- Fix: [#11512] Coloured usernames by group on multiplayer servers.
```

Comment:
This is a feature, rather than a bugfix. (Keep in mind that features are placed higher in the changelog as well.)

Reply:
janisozaur told us on the Discord that "Fix" is fitting here because there is a prior issue filed that our pr solves